library/src/main/java/com/bumptech/glide/request/RequestOptions.java

```diff
@@ -877,11 +878,36 @@ public RequestOptions transform(@NonNull Transformation<Bitmap> transformation)
   }
 
   /**
+   * Applies the given {@link Transformation}s in the given order for
+   * {@link Bitmap Bitmaps} to the default types ({@link Bitmap},
+   * {@link android.graphics.drawable.BitmapDrawable}, and
+   * {@link com.bumptech.glide.load.resource.gif.GifDrawable})
+   * and throws an exception if asked to transform an unknown type.
+   * <p>
+   * <p>This will override previous calls to {@link #dontTransform()}.
+   *
+   * @param transformations One or more {@link Transformation}s for
+   *                        {@link Bitmap}s.
+   * @see #optionalTransform(Transformation)
+   * @see #optionalTransform(Class, Transformation)
+   */
+  public RequestOptions multiTransform(@NonNull Transformation<Bitmap>... transformations) {
```

Comment:
Well, it closer resembled the name of the class that actually does the transformation, but I have no strong obligations there, so I will rename it.